docs/guides/authentication/index.md

```diff
@@ -23,7 +23,7 @@ code when it needs to find out about the user's credentials.
 
 The callback is provided via the `credentials` field in the
 `git_remote_callbacks` structure. This structure is embedded in both
-`git_fetch_options` and `git_push_options` so you can specify it both
+`git_fetch_options` and `git_push_options` so you can specify it
 both operations.
```

Comment:
This still parses a bit odd.  How about:

> The callback is provided via the `credentials` field in the
> `git_remote_callbacks` structure. This structure is embedded in both
> `git_fetch_options` and `git_push_options` so you can specify it
> in both operations.

Reply:
"to both operations"?  Either way, it needs a preposition, I think.

Reply:
I think "in" sounds better here personally